appserver/tests/tck/jsonp/pom.xml

```diff
@@ -0,0 +1,179 @@
+
+    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.glassfish.main.tests.tck</groupId>
+        <artifactId>tck</artifactId>
+        <version>7.0.15-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>glassfish-external-tck-jsonp</artifactId>
+
+    <name>Jakarta JSON Processing TCK run on GlassFish</name>
+    <description>This verifies the compliance of Eclipse Glassfish using the Jakarta JSON Processing standalone TCK</description>
+
+    <properties>
+        <maven.compiler.release>11</maven.compiler.release>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+        <tck.version>2.1.1</tck.version>
+
+        <glassfish.version>${project.version}</glassfish.version>
+        <glassfish.root>${project.build.directory}</glassfish.root>
+        <glassfish.home>${glassfish.root}/glassfish7</glassfish.home>
+        <glassfish.artifact.id>glassfish</glassfish.artifact.id>
+
+        <skip.surefire.report>true</skip.surefire.report>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.junit</groupId>
+                <artifactId>junit-bom</artifactId>
+                <version>${junit.version}</version>
+                <scope>import</scope>
+                <type>pom</type>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>jakarta.json</groupId>
+            <artifactId>jakarta.json-tck-common</artifactId>
+            <version>${tck.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>jakarta.json</groupId>
+            <artifactId>jakarta.json-tck-tests</artifactId>
+            <version>${tck.version}</version>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>jakarta.json</groupId>
+                    <artifactId>jakarta.json-api</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.netbeans.tools</groupId>
```

Comment:
Should maybe use the newer jakarta tck version.

```xml
<groupId>jakarta.tck</groupId>
<artifactId>sigtest-maven-plugin</artifactId>
```

Reply:
With Jakarta's plugin I'm getting:
```
[ERROR] Errors: 
[ERROR]   JSONPSigTest.signatureTest:281 Fault JSONPSigTest.signatureTest() failed!, diffs found
```

This is, at least, interesting...

Reply:
I missed that before - this `sigtest-maven-plugin` is declared as dependency here. Is it used as plugin somewhere?

Reply:
> I missed that before - this `sigtest-maven-plugin` is declared as dependency here. Is it used as plugin somewhere?

Not used. It's called directly from code:

https://github.com/jakartaee/jsonp-api/blob/21733cff9ddf08338710d87d23739c9c3276553f/tck/tck-tests/src/main/java/ee/jakarta/tck/jsonp/signaturetest/SigTestDriver.java#L163

Reply:
> Should maybe use the newer jakarta tck version.

> This is, at least, interesting...

Works with `jakarta.tck:sigtest-maven-plugin:[2.0-2.2]` but failed with `jakarta.tck:sigtest-maven-plugin:2.3`